src/main/webapp/app/exam/participate/summary/exam-participation-summary.component.ts

```diff
@@ -100,25 +146,15 @@ export class ExamParticipationSummaryComponent implements OnInit {
      * returns the students' submission for the exercise, undefined if no participation could be found
      */
     getSubmissionForExercise(exercise: Exercise) {
-        if (
-            exercise &&
-            exercise.studentParticipations &&
-            exercise.studentParticipations.length > 0 &&
-            exercise.studentParticipations[0].submissions &&
-            exercise.studentParticipations[0].submissions.length > 0
-        ) {
-            return exercise.studentParticipations[0].submissions[0];
-        }
+        return exercise?.studentParticipations?.[0]?.submissions?.[0];
     }
 
     /**
      * @param exercise
      * returns the students' submission for the exercise, undefined if no participation could be found
      */
     getParticipationForExercise(exercise: Exercise) {
-        if (exercise.studentParticipations && exercise.studentParticipations[0]) {
-            return exercise.studentParticipations[0];
-        }
+        return exercise.studentParticipations?.[0];
```

Comment:
same comment as above

Reply:
See my other comments. The function was implicitly undefined, which is not a good practice in general IMHO. Do we have [consistent-return](https://eslint.org/docs/rules/consistent-return) checking enabled?

Reply:
I talked with @bassner about the optional chaining related changes and I am forwarding relevant parts of his findings here:

> Your code is 99% identical to the old one. If you want it to be 100%, do return `exercise.studentParticipations?.[0] || undefined;`

Reply:
Added `|| undefined` to make it the same as the old version and added unit tests for this simplification at commit https://github.com/ls1intum/Artemis/pull/4932/commits/9896b87f6e9769b29ac20b428dd0bfcf84dc56d2.